homeassistant/bootstrap.py

```diff
@@ -186,6 +186,24 @@ def from_config_file(config_path, hass=None):
 def enable_logging(hass):
     """ Setup the logging for home assistant. """
     logging.basicConfig(level=logging.INFO)
+    fmt = ("%(log_color)s%(asctime)s %(levelname)s (%(threadName)s) "
+           "[%(name)s] %(message)s%(reset)s")
+    try:
+        from colorlog import ColoredFormatter
+        logging.getLogger().handlers[0].setFormatter(ColoredFormatter(
+            fmt,
+            datefmt='%y-%m-%d %H:%M:%S',
+            reset=True,
+            log_colors={
+                'DEBUG': 'cyan',
+                'INFO': 'green',
+                'WARNING': 'yellow',
+                'ERROR': 'red',
+                'CRITICAL': 'red',
+            }
+        ))
+    except ImportError:
+        _LOGGER.warn("No colorlog package found")
```

Comment:
Can we change this message to be a bit more descriptive, something like:
colorlog package not found, console coloring disabled.